Resource name constraints (3) (#19120)
kubernetes/website: content/en/docs/tasks/run-application/horizontal-pod-autoscale.md

can be found in `autoscaling/v2beta2`. The new fields introduced in `autoscaling/v2beta2`
are preserved as annotations when working with `autoscaling/v1`.

> When creating an `HorizontalPodAutoscaler` API object, make sure the name specified is a valid

Reviewer: nit:
```suggestion
When you create a HorizontalPodAutoscaler API object, make sure the name specified is a valid
```
